tests/h/conftest.py

```diff
@@ -28,6 +28,7 @@
                                                 'postgresql://postgres@localhost/htest'))
 
 Session = sessionmaker()
+authority = 'example.com'
```

Comment:
Given this isn't expected to change, it should probably be a "constant": `TEST_AUTHORITY = 'example.com'`